test/unit/utils/caseTabTypeHelper.js

```diff
@@ -58,6 +58,7 @@ const nonProdTabIds = [
   // 'SolicitorCoRespondent', - TODO - uncomment this when we go live with AOS pack 2
```

Comment:
Could we uncomment this?

Reply:
I have no idea why we have this file.

If it's still commented out apparently we don't need it.